client/gutenberg/editor/edit-post/components/header/index.js

```diff
@@ -58,9 +57,6 @@ function Header( {
 						isToggled={ isEditorSidebarOpened }
 						aria-expanded={ isEditorSidebarOpened }
 					>
-						<DotTip id="core/editor.settings">
```

Comment:
This was causing the editor to break, so I'm removing it since this is the only usage of `DotTip` in edit post, and we don't want to use the guided tour anyway. I'm suspecting its related to some specific error with `@wordpress/nux` package and not to our integration.